.gitignore

```diff
@@ -16,3 +16,4 @@
 
 # Ignore the generated pact
 spec/pacts/publishing_api-content_store.json
+.DS_Store
```

Comment:
I have added this to the `gitignore` in the past and I've been told that as because not every contributor uses a machine running Apple OS X, the `.DS_Store` should be ignored locally in the global `gitignore` file. 

http://technicalpickles.com/posts/configure-git-to-globally-ignore-some-files/